superset-frontend/src/SqlLab/components/ScheduleQueryButton.jsx

```diff
@@ -92,6 +92,10 @@ const defaultProps = {
   tooltip: null,
 };
 
+const StyledRow = styled(Row)`
+  padding-bottom: ${({ theme }) => theme.gridUnit * 2.5}px;
```

Comment:
It's splitting hairs, but in a case like this, we could probably round to the nearest gridUnit. In this case, I'm _guessing_ 2 would look just fine. Let me know if you think that's not the case :D

Reply:
Yes, that's a good idea - both options look fine, but let's round to the nearest gridUnit.